extension/src/utilities.rs

```diff
@@ -110,4 +149,28 @@ mod tests {
             assert!((test_val - -42f64).abs() < f64::EPSILON);
         });
     }
+
+    #[pg_test]
+    fn test_days_in_month() {
+        Spi::execute(|client| {
+            let test_val = client.select("SELECT toolkit_experimental.days_in_month('2021-01-01 00:00:00+03'::timestamptz)",None,None,).first().get_one::<i64>().unwrap();
+            assert_eq!(test_val, 31);
+        });
+
+        Spi::execute(|client| {
+            let test_val = client.select("SELECT toolkit_experimental.days_in_month('2020-02-03 00:00:00+03'::timestamptz)",None,None,).first().get_one::<i64>().unwrap();
+            assert_eq!(test_val, 29);
+        });
+    }
+    #[pg_test]
+    fn test_monthly_normalize() {
+        Spi::execute(|client| {
+            let test_val = client.select("SELECT toolkit_experimental.month_normalize(1000,'2021-01-01 00:00:00+03'::timestamptz)",None,None,).first().get_one::<f64>().unwrap();
+            assert_eq!(test_val, 981.8548387096774f64);
+        });
+        Spi::execute(|client| {
+            let test_val = client.select("SELECT toolkit_experimental.month_normalize(1000,'2021-01-01 00:00:00+03'::timestamptz,30.5)",None,None,).first().get_one::<f64>().unwrap();
```

Comment:
I'd want to test with a value like 30 to make sure we don't accidentally do integer arithmetic on this. (Could just add one more test case).